pom.xml

```diff
@@ -525,9 +525,17 @@
               <configuration>
                 <versionFormat>^\d+\.\d+\.\d+$</versionFormat>
                 <failSeverity>breaking</failSeverity>
+                <checkDependencies>false</checkDependencies>
                 <analysisConfigurationFiles>
                   <file>${project.basedir}/revapi.json</file>
                 </analysisConfigurationFiles>
+                <analysisConfiguration>
+                  <revapi.java>
+                    <missing-classes>
```

Comment:
Wasn't this already active or a new rule?

Reply:
Let me try to remove it - that was an approach to fix class loading errors (which do not happen with 17).